src/cache_manager.cc

```diff
@@ -234,7 +227,7 @@ CacheManager::ParseHeaders(const HttpRequest * request, Mgr::ActionParams &param
     // TODO: use the authentication system decode to retrieve these details properly.
 
     /* base 64 _decoded_ user:passwd pair */
-    const auto basic_cookie(request->header.getAuthToken(Http::HdrType::AUTHORIZATION, "Basic"));
+    const auto basic_cookie(request->header.getAuthToken(Http::HdrType::PROXY_AUTHORIZATION, "Basic"));
```

Comment:
I am concerned that this will break working non-squidclient use cases that use Authorization header. It is also difficult (for me) to say which header cache manager requests _should_ use. How about getting the cookie from Authorization (if that header is present) and from Proxy-Authorization (otherwise)?

Reply:
>  getting the cookie from Authorization (if that header is present) and from Proxy-Authorization (otherwise)

Done.